### PR TITLE
import: add `create_microarray_annotation()` method

### PR DESCRIPTION
**Pull request by klas-genestack**
refs #6087

### CODE REVIEW

**Comment by klas-genestack** [comment]
I've bumped major version because of new functionality.

File: genestack_client/data_importer.py [hunk 1]
```diff
@@ -55,6 +55,19 @@ class DataImporter(object):
...
     If you are uploading a local file, a ``Raw Upload`` intermediary file will be created on the platform.
     """
+
+    # FIXME: make them appear in Sphinx docs
+    #: Affymetrix microarray annotation type
+    AFFYMETRIX_ANNOTATION = 'affymetrixMicroarrayAnnotation'
+    #: Agilent microarray annotation type
+    AGILENT_ANNOTATION = 'agilentMicroarrayAnnotation'
+
+    #: List of available microarray annotation types
```
Comment (genestack-solomatin) [inline genestack_client/data_importer.py]: List :)

File: genestack_client/data_importer.py [hunk 2]
```diff
@@ -55,6 +55,19 @@ class DataImporter(object):
 
     If you are uploading a local file, a ``Raw Upload`` intermediary file will be created on the platform.
     """
+
+    # FIXME: make them appear in Sphinx docs
```
Comment (genestack-solomatin) [inline genestack_client/data_importer.py]: yes, please

File: genestack_client/data_importer.py [hunk 3]
```diff
@@ -562,3 +575,35 @@ def create_owl_ontology(self, parent=None, name=None, url=None, metainfo=None):
...
+    def create_microarray_annotation(self, annotation_type, parent=None,
+                                     name=None, url=None, metainfo=None):
+        """
+        Create a Dictionary file from a local or remote microarray annotation
+        file.
+        ``name`` and ``url`` are required fields.
+        They can be specified through the arguments or
+        via a :py:class:`~genestack_client.BioMetainfo` instance.
+
+        :param annotation_type: type of loaded annotation
```
Comment (genestack-solomatin) [inline genestack_client/data_importer.py]: may be add couple of words that not any string is suitable?

File: genestack_client/data_importer.py [hunk 4]
```diff
@@ -562,3 +574,36 @@ def create_owl_ontology(self, parent=None, name=None, url=None, metainfo=None):
         name and metainfo.add_string(BioMetainfo.NAME, name)
         url and metainfo.add_external_link(BioMetainfo.DATA_LINK, url)
         return self.__invoke_loader(parent, 'dictionaryFiles', metainfo)
+
+    def create_microarray_annotation(self, annotation_type, parent=None,
+                                     name=None, url=None, metainfo=None):
+        """
+        Create a Dictionary file from a local or remote microarray annotation
+        file.
```
Comment (genestack-solomatin) [inline genestack_client/data_importer.py]: move to previous line?

**Pull request merged**